README.md

```diff
@@ -1,26 +1,52 @@
-# jwt-auth-bundle
+![jwt-auth-bundle](.github/header.png)
```

Comment:
I uploaded this to the CDN so that it works across any site that might embed the readme
```suggestion
![jwt-auth-bundle](https://cdn.auth0.com/website/sdks/banners/jwt-auth-bundle-banner.png)
```